sky/clouds/gcp.py

```diff
@@ -287,10 +294,10 @@ def make_deploy_resources_variables(
 
         # gcloud compute images list \
         # --project deeplearning-platform-release \
-        # --no-standard-images
+        # --no-standard-images | grep ubuntu-2004
         # We use the debian image, as the ubuntu image has some connectivity
         # issue when first booted.
```

Comment:
remnant?

Reply:
Since we are using ubuntu image now, we use `grep ubuntu-2004` to find the correct images.